docs/configure-renovate-ce-github.md

```diff
@@ -77,6 +79,13 @@ Mend Renovate requires configuration via environment variables in addition to Re
 
 **`GITHUB_COM_TOKEN`**: A Personal Access Token for a user account on github.com (i.e. _not_ an account on your GitHub Enterprise instance). This is used for retrieving changelogs and release notes from repositories hosted on github.com and it does not matter who it belongs to. It needs only read-only access privileges. Note: This is required if you are using a self-hosted GitHub Enterprise or GitLab instance but should not be configured if your `RENOVATE_ENDPOINT` is `https://api.github.com`.
 
+**`MEND_RNV_AUTODISCOVER_FILTER`**: a string of a comma separated values (e.g. `org1/*, org2/test*, org2/test*`)
```

Comment:
Clarify that it's the same functionality as in Renovate

Reply:
Does it work exactly the same as in renovate, meaning it only affects the autodiscovery on all platforms, but webhooks too on GitLab, or is it implemented differently? 

If it also affects webhooks on GitHub, this would resolves #303 directly at least for the use case I opened the issue for.

Reply:
Can you please also add a clarification that the `autodiscoverFilter` configuration of renovate should be turned off completely when using renovate-(ce|ee), please?

I'm sure I won't be the last one confused by that.

Reply:
FYI we are adding code to ensure autodiscover is always forced off once the config is passed to Renovate, although it would still be good to clarify this as you ask

Reply:
update

Reply:
> Can you please also add a clarification that the autodiscoverFilter configuration of renovate should be turned off completely when using renovate-(ce|ee), please?

Can be handled in a separate PR because I'm not sure where should this message be added to since it true for all platforms and editions

Reply:
> Does it work exactly the same as in renovate, meaning it only affects the autodiscovery on all platforms, but webhooks too on GitLab, or is it implemented differently?

implementation is platform agnostic; repositories that dont match any filter wont be included in the database and wont be scanned (webhook included).

Reply:
@Gabriel-Ladzaretti Thank you, that is great! We were maintaining our own small service to filter the webhooks so far, this means we can just get rid of that completely.